server/Makefile

```diff
@@ -457,7 +457,7 @@ test-server: export GOTESTSUM_FORMAT := $(GOTESTSUM_FORMAT)
 test-server: export GOTESTSUM_JUNITFILE := $(GOTESTSUM_JUNITFILE)
 test-server: export GOTESTSUM_JSONFILE := $(GOTESTSUM_JSONFILE)
 test-server: test-server-pre
-	$(GOBIN)/gotestsum --rerun-fails=3 --packages="$(TE_PACKAGES) $(EE_PACKAGES)" -- $(GOFLAGS)-timeout=90m
+	$(GOBIN)/gotestsum --rerun-fails=3 --packages="$(TE_PACKAGES) $(EE_PACKAGES)" -- $(GOFLAGS) -timeout=90m
```

Comment:
Unrelated change

Reply:
Yes totally unrelated, there's another one in server/cmd/mmctl/commands/channel_test.go which is a typo fix but I thought there were not worth a dedicated PR. Should I split at least that one in another PR ?

Reply:
Would you mind opening a separate PR for the typo fix and this one? That way, we keep the history clean.

Reply:
Sure

Reply:
Here it is https://github.com/mattermost/mattermost/pull/28710